update(LDAP authentication): Add troubleshooting tips

ugaston opened the pull request:
Versions: 7.9, 7.10, 7.11, 7.12

Add new troubleshooting information.

ugaston committed:
update(LDAP authentication): Add troubleshooting tips

Versions: 7.9, 7.10, 7.11, 7.12

Add new troubleshooting information.

alachambre commented on modules/ROOT/pages/active-directory-or-ldap-authentication.adoc:
```diff
@@ -259,14 +259,86 @@ The Active Directory configuration has been tested in single domain configuratio
+* Enable JAAS debug mode by editing your JAAS configuration file (e.g. `<TOMCAT_HOME>/server/conf/jaas.cfg`) and adding the following line: `debug=true`
+* Enable java's https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ReadDebug.html[SSL/TLS connection debug mode] by editing your application server's environment setting file (e.g. `<TOMCAT_HOME>/setup/tomcat-templates/setenv.sh(.bat)`) and adding the following system property to the java options: `-Djavax.net.debug=all`
+
+[discrete]
```
```suggestion
```

I suggest you to never use [discrete], it removes the creation of the anchor and the section doesn't appear in the table of content. I can see many [discrete] tags in this page (probably something that remain from the conversion from markdown), you could maybe remove them all.

alachambre commented on modules/ROOT/pages/active-directory-or-ldap-authentication.adoc:
```diff
@@ -259,14 +259,86 @@ The Active Directory configuration has been tested in single domain configuratio
+* Enable java's https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ReadDebug.html[SSL/TLS connection debug mode] by editing your application server's environment setting file (e.g. `<TOMCAT_HOME>/setup/tomcat-templates/setenv.sh(.bat)`) and adding the following system property to the java options: `-Djavax.net.debug=all`
+
+[discrete]
+==== Common error examples
```
```suggestion
=== Common error examples
```

And you should not jump a title level, it can lead to a confusing rendering.

ugaston committed:
Update modules/ROOT/pages/active-directory-or-ldap-authentication.adoc

Co-authored-by: Adrien <adrien.lachambre@bonitasoft.com>

ugaston committed:
Update modules/ROOT/pages/active-directory-or-ldap-authentication.adoc

Co-authored-by: Adrien <adrien.lachambre@bonitasoft.com>

ugaston committed:
fix(LDAP authentication): fix content format + others

- Remove [discrete] tags
- Add a couple of troubleshooting tips
- Try to improve the content format

ugaston committed:
fix(LDAP authentication): Add table frame and grid

That's it.
